tests/test_evaluators.py

```diff
@@ -482,3 +482,104 @@ def test_evaluation_structure(self):
         )
         self.assertIsInstance(eval_results, RankBasedMetricResults)
         assert eval_results.arithmetic_mean_rank == self.counter, 'Should end at the same value as it started'
+
+
+class DummyFilterEvaluator(Evaluator):
```

Comment:
put this in `tests/mocks`

Reply:
Thanks for pointing this out. I don't even need it anymore 😅

Reply:
What about the other Dummy that was there before?

Reply:
never got around to moving it, I didn't want to make a bigger diff than I normally cause